src/util/slateHelpers.js

```diff
@@ -68,6 +67,42 @@ const RULES = [
   {
     // Aside handling
     deserialize(el, next) {
+      if (el.tagName.toLowerCase() !== 'div') return;
```

Comment:
Ikke et must, men hadde vært fint med tester for deserialize og serialize.

Reply:
Fikset